scripts/upload-data.js

```diff
@@ -256,48 +304,61 @@ const main = async () => {
           type: "number",
         },
       ],
-    },
-    {
-      filePath: locationsfile,
+    });
+  }
+
+  // barriers schemas for hospitalization hotspots
+  if (id === "hospitalization_hotspots") {
+    files.push({
+      filePath: statebarriersfile,
       extension: "json",
-      field: "locations",
-      isArray: true,
+      field: "state_barriers",
+      isArray: false,
```

Comment:
I think it still is an array?

Reply:
No! Already updated, i just forgot to commit it 🤦